sw/airborne/modules/nav/nav_smooth.c

```diff
@@ -160,22 +160,22 @@ static inline float ground_speed_of_course(float x)
 /* Compute the ground speed for courses 0, 360/NB_ANGLES, ...
    (NB_ANGLES-1)360/NB_ANGLES */
 static void compute_ground_speed(float airspeed,
-                                 float wind_x,
-                                 float wind_y)
+                                 float wind_east,
+                                 float wind_north)
 {
   uint8_t i;
   float alpha = 0;
-  float c = wind_x * wind_x + wind_y * wind_y - airspeed * airspeed;
+  float c = wind_north * wind_north + wind_east * wind_east - airspeed * airspeed;
   for (i = 0; i < NB_ANGLES; i++, alpha += ANGLE_STEP) {
     /* g^2 -2 scal g + c = 0 */
-    float scal = wind_x * cos(alpha) + wind_y * sin(alpha);
+    float scal = wind_north * cos(alpha) + wind_east * sin(alpha);
```

Comment:
Interesting - I renamed it in Eclipse but apparently this one wasn't renamed properly. Fixed.